docs/src/content/docs/fr/resources/community-content.mdx

```diff
@@ -127,24 +127,27 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
 		},
 		{
 			href: 'https://www.youtube.com/watch?v=-Ki-1E5gNCk',
-			title:
-				"Modèle de documentation Astro Starlight (créer des documentations personnalisées d'application !)",
+			title: "Modèle de documentation Astro Starlight (créer des documentations personnalisées d'application !)",
 			description:
 				"Créez un nouveau site Starlight prêt à l'emploi en environ 5 minutes",
 		},
 		{
 			href: 'https://www.youtube.com/watch?v=12o7WxjAxjM',
-			title:
-				'Inclure une documentation Starlight dans un projet Next.js avec des proxys',
+			title: 'Inclure une documentation Starlight dans un projet Next.js avec des proxys',
 			description:
 				"Configurer un projet Starlight dans un sous-répertoire au sein d'un site Next.js",
 		},
 		{
 			href: 'https://www.youtube.com/watch?v=5pq80drDrNs',
-			title:
-				"J'ai recréé la documentation de Shadcn en 30 minutes avec cet outil incroyable",
+			title: "J'ai recréé la documentation de Shadcn en 30 minutes avec cet outil incroyable",
 			description:
 				"Dans cette vidéo, je passe en revue ce qui rend Starlight si cool et pourquoi vous voudriez l'essayer avec votre prochain projet.",
 		},
+		{
+			href: 'https://www.youtube.com/watch?v=Q1E4Gkt63ko',
+			title: "Astro Starlight et emoji-blast avec Chris Swithinbank",
+			description:
+				"Chris, le créateur du framework Astro's Starlight, explique comment le mettre en place pour un nouveau site web d'emoji-blast. 🌟 🎇",
```

Comment:
```suggestion
				"Chris, le créateur du thème de documentation Starlight pour Astro, m'aide à le configurer pour le nouveau site web d'emoji-blast. 🌟 🎇",
```

The English `'s` in `Astro's Starlight` was remaining so I tweaked the sentence a bit to remove it and uses the same description ("thème de documentation") that we are using on the French [version](https://starlight.astro.build/getting-started/) of the getting started guide.